### ISSUE TITLE
When hint is not set, vertical spacing changes when there are no validation errors

### ISSUE THREAD

**Issue by Al-Dyachkov** [AvaloniaCommunity/Material.Avalonia#167]
Same card with validation errors and with no validation errors.

![image](https://user-images.githubusercontent.com/10960735/164947764-dc93e7b9-93f4-4311-bb7d-80cade24dc08.png)

Can be worked around with setting single space as hint when it's not required, but I'd rather had that space reserved for validation errors automatically.

```xaml
  assists:TextFieldAssist.Hints=" "
```

**Comment by SKProCH** [comment]
Hello, @Al-Dyachkov 
As far as I remember, the long error text will be divided into several lines if necessary. We can reserve the height, but you will still need more space in the case of long errors. Does this make sense?

**Comment by Al-Dyachkov** [comment]
Yes, it does. I'm ok with explicitly adding space for long errors.

**Comment by appleneko2001** [comment]
Hi @Al-Dyachkov !

I could provide a style patch for you, just simply add this to your `App.Styles` section in your App.axaml

```
<Style Selector="TextBox /template/ TextBlock#HintsTextBlock">
    <Setter Property="TextWrapping" Value="Wrap"/>
</Style>
```
this patch supports only before 3.0 version, library is allow use styles to change their controls look. I might want add this thing as example to wiki

![image](https://user-images.githubusercontent.com/38249047/178192260-5100dcee-987d-44e4-a85c-d259bbc03c4b.png)
![image](https://user-images.githubusercontent.com/38249047/178192417-d87c1cbd-8a37-4cb8-b7d0-752bb656422c.png)
![image](https://user-images.githubusercontent.com/38249047/178192428-808467cc-ecea-4674-b628-53c243f7f12d.png)
